docs/components/modeler/bpmn/message-events/message-events.md

```diff
@@ -18,7 +18,7 @@ When the message subscription is created, a message can be correlated to the sta
 
 Messages are **not** correlated if they were published before the process was deployed or if a new version of the process is deployed without a proper start event.
 
-The `correlationKey` of a published message can be used to control the process instance creation. If an instance of this process is active (independently from its version) and it was triggered by a message with the same `correlationKey`, the message is **not** correlated and no new instance is created.
+The `correlationKey` of a published message can be used to control the process instance creation. If an instance of this process is active (independently from its version) and it was triggered by a message with the same `correlationKey`, the message is **not** correlated and no new instance is created. New messages will be queued.
```

Comment:
:wrench: The term queue is not used on this page. Instead, the message docs use the term buffering.

What do you think about the following? 

```suggestion
The `correlationKey` of a published message can be used to control the process instance creation. If an instance of this process is active (independently from its version) and it was triggered by a message with the same `correlationKey`, the message is **not** correlated and no new instance is created. If the message has a time-to-live (TTL) > 0, it is buffered.
```